crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -523,3 +523,8 @@ impl From<&Owner> for OwnerImpl {
         }
     }
 }
+
+#[derive(Clone, Debug, Union)]
```

Comment:
nit: can you move this up top with the other types and add a doc comment explaining what it is? (The doc comments feed into schema generation, so the intended audience is someone using the GraphQL API, rather than someone working on the GraphQL service codebase).

Reply:
Sure, done